src/cli/subcommand/repair.rs

```diff
@@ -21,40 +34,472 @@ pub struct Repair {
+                        repair_actions.push(reconfigure);
+                    },
+                    _ => {
+                        // Linux-specific hook repair actions, once we have them
+                    },
+                }
+
+                None
+            },
+            RepairKind::Sequoia {
+                nix_build_user_prefix,
+                nix_build_user_count,
+                nix_build_user_id_base,
+                nix_build_group_name,
+                move_existing_users,
+            } => {
+                if !matches!(
+                    OperatingSystem::host(),
+                    OperatingSystem::MacOSX { .. } | OperatingSystem::Darwin
+                ) {
+                    return Err(color_eyre::eyre::eyre!(
+                        "The `sequoia` repair command is only available on macOS"
+                    ));
+                }
+
+                if !std::io::stdin().is_terminal() && !self.no_confirm {
+                    return Err(color_eyre::eyre::eyre!(
+                        "The `sequoia` repair command should be run in an interactive terminal. If \
+                        you accept the risks of an unattended repair, pass `--no-confirm`."
+                    ));
+                }
+
+                let existing_receipt = get_existing_receipt().await;
+
+                let maybe_create_users_and_groups_idx_action =
+                    find_users_and_groups(existing_receipt)?;
+
+                let (
+                    user_prefix,
+                    user_count,
+                    user_base,
+                    group_name,
+                    group_gid,
+                    receipt_action_idx_create_group,
+                ) = match maybe_create_users_and_groups_idx_action {
+                    Some((receipt, create_users_and_groups_idx, action)) => {
+                        tracing::debug!("Found {} in receipt", CreateUsersAndGroups::action_tag());
+
+                        let user_prefix = action.nix_build_user_prefix;
+                        let user_count = action.nix_build_user_count;
+                        let user_base = crate::settings::default_nix_build_user_id_base();
+                        let group_gid = action.nix_build_group_id;
+                        let group_name = action.nix_build_group_name;
+
+                        (
+                            user_prefix,
+                            user_count,
+                            user_base,
+                            group_name,
+                            Some(group_gid),
+                            Some((receipt, create_users_and_groups_idx, action.create_group)),
+                        )
+                    },
+                    None => {
+                        tracing::warn!(
+                            "Unable to find {} in receipt (receipt didn't exist or is incompatible \
+                            with this version of the installer), not updating receipt with migrated \
```

Comment:
```suggestion
                            with this version of the installer), not updating receipt with migrated \
```

Can you make this a nicer set of set of sentences targeting a human? Describe clearly what won't happen (receipt updating) and also what will happen (users will be relocated)